src/main/java/it/gov/pagopa/reward/notification/controller/NotificationController.java

```diff
@@ -108,4 +105,10 @@ Mono<ResponseEntity<Void>> suspendUserOnInitiative(
             @PathVariable("organizationId") String organizationId,
             @PathVariable("initiativeId") String initiativeId,
             @PathVariable("userId") String userId);
+
+    @DeleteMapping(value = "/organization/{organizationId}/initiative/{initiativeId}/user/{userId}/readmit")
```

Comment:
I will not use DELETE with a separate URL...
It seems that you are deleting a "readmit" request...
just use a POST or PUT